File: 1-two-sum/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {

        for ( int i=0 ; i<nums.size();i++){
        int temp = target - nums[i];

        auto it = find(nums.begin() + i + 1,nums.end(),temp);

        if ( it != nums.end()){
                return { i , static_cast<int>(distance(nums.begin(),it))};

             }
        }

       return {0}; 
    }
};

File: 1-two-sum/two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        temp=dict()
        for i in range(len(nums)):
            if target-nums[i] in temp:
                return temp[target-nums[i]],i
            temp[nums[i]]=i
        return []
        